sdk/src/main/java/io/verloop/sdk/Verloop.kt

```diff
@@ -118,7 +112,6 @@ class Verloop(val context: Context, var verloopConfig: VerloopConfig) {
             private const val TAG = "VerloopInterface"
         }
 
-        @JavascriptInterface
```

Comment:
Don't we need these annotations anymore?

Reply:
No, it is not required.